### PR TITLE
Typo cleanup for app

### PR DESCRIPTION
**Pull request by akiojalehto**
### Description
I was adjusting my code inspection and made a little larger cleanup for typos while at it. No breaking changes, just misspellings.

### Contribution checklist
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [X] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Review by ishakhsuvarov — APPROVED** [review]
Changes look good to me. I am going to invite more people for the review just in case, as change set is rather big.

**Comment by ishakhsuvarov** [comment]
@akiojalehto Thank you for contribution. Changes look good to me.
I had to add a suppress warning in one place as tests detect issues not related to this PR (Static tests are executed only on the files changed).

**Comment by akiojalehto** [comment]
@ishakhsuvarov Great, thank you. I noticed that the test result was unrelated to PR, but didn't spot the logic for it.

File: app/code/Magento/Catalog/Api/Data/ProductRender/PriceInfoInterface.php [hunk 1]
```diff
@@ -51,7 +51,7 @@ public function setMaxPrice($maxPrice);
 
     /**
      * Set max regular price
-     * Max regular price is the same, as maximum price, except of excluding calculating special price and catalogules
+     * Max regular price is the same, as maximum price, except of excluding calculating special price and catalogues
```
Comment (orlangur) [inline app/code/Magento/Catalog/Api/Data/ProductRender/PriceInfoInterface.php]: catalog rules maybe?

Comment (ishakhsuvarov) [reply app/code/Magento/Catalog/Api/Data/ProductRender/PriceInfoInterface.php]: Looks like yes

**Comment by ishakhsuvarov** [comment]
@akiojalehto I took the liberty of updating the doc block with more appropriate wording. Hope you don't mind

**Comment by akiojalehto** [comment]
@ishakhsuvarov Don't mind at all, it's much faster if you guys update directly all the things you see more appropriate.

**Pull request merged**